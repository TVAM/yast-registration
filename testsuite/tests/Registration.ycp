{

// testedfiles: Registration.ycp

include "testsuite.ycp";
TESTSUITE_INIT([], nil);

import "Registration";

DUMP("Registration::Modified");
TEST(``(Registration::Modified()), [], nil);

}
