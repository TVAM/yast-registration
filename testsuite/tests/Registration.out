Dump	Registration::Modified
Return	false
